tools/vipps-api-global-postman-environment.json

```diff
@@ -0,0 +1,364 @@
+		},
+		{
+			"key": "client_secret",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Client secret here>",
+			"enabled": true
+		},
+		{
+			"key": "merchantSerialNumber",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Copy the Merchant Serial Number from the Sale Unit you copied the keys from>",
+			"enabled": true
+		},
+		{
+			"key": "Ocp-Apim-Subscription-Key",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Vipps-Subscription-Key>",
+			"enabled": true
+		},
+		{
+			"key": "amount",
+			"value": "499",
+			"enabled": true
+		},
+		{
+			"key": "amount_capture",
+			"value": "0",
+			"enabled": true
+		},
+		{
+			"key": "mobileNumber",
+			"value": "<Input the mobile number for the test app profile you have received or registered>",
+			"enabled": true
+		},
+		{
+			"key": "callbackPrefix",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "fallBack",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "shippingDetailsPrefix",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "consentRemovalPrefix",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "base_url",
+			"value": "https://apitest.vipps.no",
+			"enabled": true
+		},
+		{
+			"key": "base_url_production",
+			"value": "<For Vipps-Partner-API. Use https://api.vipps.no>",
+			"enabled": true
+		},
+		{
+			"key": "access_token",
+			"value": "<This value will be overwritten when you call the Get Access Token API when having selected eCommerce v2 - MTE environment>",
+			"enabled": true
+		},
+		{
+			"key": "orderId",
+			"value": "<This value will be overwritten on each Initiate Payment call with MSN + timestamp + random int>",
+			"enabled": true
+		},
+		{
+			"key": "vippsLandingPageURL",
+			"value": "<This value will be overwritten on each Initiate Payment call. It will contain a url to the Vipps landing page, obtained from the eCom api.>",
+			"enabled": true
+		},
+		{
+			"key": "shippingCost1",
+			"value": "30",
+			"enabled": true
+		},
+		{
+			"key": "shippingCost2",
+			"value": "100",
+			"enabled": true
+		},
+		{
+			"key": "shippingMethod1",
+			"value": "Postman Method 1",
+			"enabled": true
+		},
+		{
+			"key": "shippingMethod2",
+			"value": "Postman Method 2",
+			"enabled": true
+		},
+		{
+			"key": "transactionTextInitiate",
+			"value": "Transaction initiated through Postman",
+			"enabled": true
+		},
+		{
+			"key": "transactionTextCapture",
+			"value": "Transaction captured through Postman",
+			"enabled": true
+		},
+		{
+			"key": "transactionTextRefund",
+			"value": "Transaction refunded through Postman",
+			"enabled": true
+		},
+		{
+			"key": "transactionTextCancel",
+			"value": "Transaction cancelled through Postman",
```

Comment:
```suggestion
			"value": "This payment was cancelled with Postman.",
```

Reply:
Nice! @cloveras  just commit as you go :-)